sci_analysis/analysis/__init__.py

```diff
@@ -62,6 +62,18 @@ def determine_analysis_type(data, other=None, groups=None):
         else:
             return Categorical(data)
 
+def output_df(statistics_list):
+    df_list = []
+    for stat_data in statistics_list:
```

Comment:
I'd probably do this with a dictionary instead of a list where the key is the title such as "Overall Statistics" or "Group Statistics". I've been using the {'name', DataFrame} output in a couple of other places in the codebase so it'll be more consistent.